gadgets/trace_lsm/gadget.yaml

```diff
@@ -3,40 +3,49 @@ description: a strace for LSM tracepoints
 homepageURL: https://inspektor-gadget.io/
 documentationURL: https://inspektor-gadget.io/docs
 sourceURL: https://github.com/inspektor-gadget/inspektor-gadget/
+datasources:
+  lsm:
+    fields:
+      timestamp_raw:
+        annotations:
+          columns.hidden: true
+      timestamp:
+        annotations:
+          template: timestamp
+      mntns_id:
+        annotations:
```

Comment:
Missing description (`Mount namespace inode id`)

Reply:
I'll ignore it, IMO this should be automatically added by Inspektor Gadget giving that we know the type of the field. (I tried to make it, but it was too much for this PR).